### PR TITLE
Add new tutorials pages for long-form guides

### PR DESCRIPTION
**Pull request by jfy133**
To close https://github.com/nf-core/nf-co.re/issues/437

Currently structured that each tutorial is one markdown file, and a tutorial 'parent' page just hosts markdown links to act as a ToC, but could be improved I guess.

I moved the `nf-core_tutorial` page inside teh `usage/` tutorials as an example.

For the rest of the march hackathon, I will work on making a step-by-step guide to making an institutional profile, and @lescai will start working on a DSL2 related tutorial to flesh this out a bit more.

### CODE REVIEW

**Review by apeltzer — APPROVED** [review]
Looks good to me - I think we can merge this and have separate PRs for the upcoming tutorials then?

**Comment by jfy133** [comment]
That's the plan! Just want an OK from @ewels as it involves website structure

File: markdown/developers/developer_tutorials.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+subtitle: In-depth step-by-step guides for getting started with nf-core
+---
+
+# Introduction
+
+This page includes step-by-step guides, walkthroughs, and general long-form introductions to various aspects of nf-core.
+
+They should generally be aimed for brand-new users or developers who are not sure where to start.
+
+# Tutorials Index
```
Comment (ewels) [inline markdown/developers/developer_tutorials.md:12]:
```suggestion
# Tutorials Index

_..none yet, please check back soon.._
```

File: markdown/usage/usage_tutorials.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+title: Usage Tutorials
+subtitle: In-depth step-by-step guides for getting started with nf-core
+---
+
+# Introduction
+
+This page includes step-by-step guides, walkthroughs, and general long-form introductions to various aspects of nf-core.
+
+They should generally be aimed for brand-new users or developers who are not sure where to start.
```
Comment (ewels) [inline markdown/usage/usage_tutorials.md:10]:
```suggestion
This page includes step-by-step guides, walkthroughs, and general long-form introductions to various aspects of nf-core, aimed for brand-new users or developers who are not sure where to start.
```

File: markdown/developers/developer_tutorials.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+---
+title: Developer Tutorials
+subtitle: In-depth step-by-step guides for getting started with nf-core
```
Comment (ewels) [inline markdown/developers/developer_tutorials.md:3]: ☝🏻

**Review by ewels — CHANGES_REQUESTED** [review]
Generally looks good, but I don't love moving the existing tutorial URL as we link to it from a lot of places.

Could you please just add a rewrite rule for it so that the old link forwards to the new one automatically?

https://github.com/nf-core/nf-co.re/blob/7307ba7363b5d4e4b2e814aef8feeee3e36be6d3/public_html/.htaccess#L5-L8

**Comment by jfy133** [comment]
@ewels added the re-write, hope that's correct.

**Review by ewels — APPROVED** [review]
Looks good!

**Pull request merged**